cql2/standard/schema/cql2.bnf

```diff
@@ -178,16 +178,16 @@ temporalFunction = "T_AFTER"
 # a Boolean value of true.  Otherwise the function returns false.
 #=============================================================================#
 arrayPredicate = arrayFunction
-                 "(" arrayExpression "," arrayExpression ")";
+                 "(" arrayOperand "," arrayOperand ")";
```

Comment:
@pvretano Now I'm confused... Maybe this is is what was contributing to my earlier confusion.

Why does this take 2 `arrayOperand`?